tests/migration_test.go

```diff
@@ -49,10 +49,10 @@ import (
 
 const (
 	migrationWaitTime = 240
-	fedoraVMSize      = "256M"
+	fedoraVMSize      = "1Gi"
 )
 
-var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system] VM Live Migration", func() {
+var _ = FDescribe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system] VM Live Migration", func() {
```

Comment:
Perhaps we should try running the whole suite. It might help to reveal the issue.

Reply:
It was a leftover from local run. Updated.